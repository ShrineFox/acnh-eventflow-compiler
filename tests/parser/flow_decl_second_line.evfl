# zzzz
flow Test():
    return
# zzzz
