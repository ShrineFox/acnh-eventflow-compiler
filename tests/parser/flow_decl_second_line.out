RootNode[name=Test, vardefs=[], out_edges=[TerminalNode]]
TerminalNode[]

